_includes/footer.html

```diff
@@ -6,7 +6,7 @@
       <h2>{{ site.title }}</h2>
       <p>{{ site.description }}</p>
       <p class="footer-govprinters">
-        Gazettes from the <a href="http://kenyalaw.org/kl/" target="_blank" rel="noopener">National Council for Law Reporting (Kenya Law)</a> are re-shared under <a href="https://web.archive.org/web/20170215233726/http://www.kenyalaw.org/kl/index.php?id=2161f" target="_blank" rel="noopener">Creative Commons Attribution - ShareAlike 4.0 International (CC BY-SA 4.0) license</a>.
+        Gazettes from the <a href="https://dds.crl.edu/crldelivery/27040" target="_blank" rel="noopener">Center for Research Libraries: Global Resource Network</a>.
```

Comment:
@andela-wcyn We should include the attribution in the config.yml as well for re-use.